curriculum/challenges/english/02-javascript-algorithms-and-data-structures/debugging/understanding-the-differences-between-the-freecodecamp-and-browser-console.english.md

```diff
@@ -9,15 +9,15 @@ forumTopicId: 301193
 <section id='description'>
 You may have noticed that some freeCodeCamp JavaScript challenges include their own console. This console behaves a little differently than the browser console you used in the last challenge.
 The following challenge is meant to highlight the main difference between the freeCodeCamp console and your browser console.
-When you run ordinary JavaScript, the browsers console will display your <code>console.log()</code> statements the exact number of times you requested.
-The freeCodeCamp console will print your <code>console.log()</code> statements for each test of that challenge, as well as one more time for any function calls that you have in your code.
-This lends itself to some interesting behavior and might trip you up in the beginning, because a logged value that you expect to see only once may print out many more times.
-If you would like to see only your single output and not have to worry about running through the test cycles, you can use <code>console.clear()</code> and check the browsers console.
+When you run ordinary JavaScript, the browser's console will display your <code>console.log()</code> statements the exact number of times it is called.
+The freeCodeCamp console will print your <code>console.log()</code> statements a short time after the editor detects a change in the script, as well as during every test.
+The freeCodeCamp console is cleared before every test. So, you may miss a log, if the tests finish too quickly.
+If you would like to see every log for every test, run the tests, and open the browser console. If you would like to use the browser console, and want it to operate the same as the freeCodeCamp console, place <code>console.clear()</code> before any other <code>console</code> calls, to clear the browser console.
```

Comment:
```suggestion
If you would like to see every log for every test, run the tests, and open the browser console. If you prefer to use the browser console, and want it to mimic the freeCodeCamp console, place <code>console.clear()</code> before any other <code>console</code> calls, to clear the browser console.
```
This approach shows only the last test, while the fCC console shows only the first test, but it's as similar as you can get.  To my mind 'mimic' gets across the fact that it looks the same, but isn't identical.  Also, I incorporated @AshNaz87's suggestion, as it avoids repetition.